packages/apps-v5/src/commands/apps/destroy.js

```diff
@@ -15,17 +15,19 @@ async function run(context, heroku) {
   await cli.confirmApp(app, context.flags.confirm, `WARNING: This will delete ${cli.color.app(app)} including all add-ons.`)
   let request = heroku.request({
     method: 'DELETE',
-    path: `/apps/${app}`
+    path: `/apps/${app}`,
   })
   await cli.action(`Destroying ${cli.color.app(app)} (including all add-ons)`, request)
 
   if (git.inGitRepo()) {
     // delete git remotes pointing to this app
     _(await git.listRemotes())
-      .filter((r) => git.gitUrl(app) === r[1] || git.sshGitUrl(app) === r[1])
-      .map((r) => r[0])
-      .uniq()
-      .forEach(git.rmRemote)
+    .filter(r => git.gitUrl(app) === r[1] || git.sshGitUrl(app) === r[1])
```

Comment:
I propose changing this rule. I think it's objectively and significantly less readable.